pkg/source/vmware/client.go

```diff
@@ -470,3 +493,9 @@ func adapterType(deviceID string) kubevirt.DiskBus {
 	}
 	return kubevirt.DiskBusSATA
 }
+
+// SanitizeVirtualMachineImport is used to sanitize the VirtualMachineImport object.
+func (c *Client) SanitizeVirtualMachineImport(vm *migration.VirtualMachineImport) error {
+	vm.Status.DefiniteVirtualMachineName = vm.Spec.VirtualMachineName
```

Comment:
we should also add a `strings.ToLower` conversion here since vmware allows camel case names